From 873a0fd9cae51f51eaa9d04a823bb34c386dacbf Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Mon, 9 Nov 2015 16:11:53 +0800
Subject: [PATCH 0661/5242] MLK-11840 video: mxsfb: change 'CONFIG_PM_RUNTIME'
 to 'CONFIG_PM'

commit  16d77047279a1e77cecb786cce07a3fcacb2d9ee from
https://source.codeaurora.org/external/imx/linux-imx.git

The 'CONFIG_PM_RUNTIME' has been eliminated and all the users
should use 'CONFIG_PM' directly. So do this change for mxsfb.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxsfb.c |   10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index 9020a6d..d86436a 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -1535,7 +1535,7 @@ static void mxsfb_shutdown(struct platform_device *pdev)
 	}
 }
 
-#ifdef CONFIG_PM_RUNTIME
+#ifdef CONFIG_PM
 static int mxsfb_runtime_suspend(struct device *dev)
 {
 	release_bus_freq(BUS_FREQ_HIGH);
@@ -1551,12 +1551,7 @@ static int mxsfb_runtime_resume(struct device *dev)
 
 	return 0;
 }
-#else
-#define	mxsfb_runtime_suspend	NULL
-#define	mxsfb_runtime_resume	NULL
-#endif
 
-#ifdef CONFIG_PM
 static int mxsfb_suspend(struct device *pdev)
 {
 	struct fb_info *fb_info = dev_get_drvdata(pdev);
@@ -1585,6 +1580,9 @@ static int mxsfb_resume(struct device *pdev)
 	return 0;
 }
 #else
+#define	mxsfb_runtime_suspend	NULL
+#define	mxsfb_runtime_resume	NULL
+
 #define	mxsfb_suspend	NULL
 #define	mxsfb_resume	NULL
 #endif
-- 
1.7.9.5

